jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -727,9 +727,9 @@ protected void deployScriptsToUserJwalaScriptsDir(Jvm jvm, User user) throws Com
 	/**
 	 * This method upgrades the JDK version for the mentioned JVM
 	 * 
-	 * @param jvmName
-	 * @param user
-	 * @return Jvm
+	 * @param jvmName - name of the JVM that needs to be upgraded

```

Comment:
The hyphen between the parameter name and the description is not needed. Look at the other javadoc comments in this class for examples of correct formatting.